testplan/web_ui/testing/src/Nav/__tests__/InteractiveNav.test.js

```diff
@@ -20,7 +20,7 @@ describe('InteractiveNav', () => {
   it('shallow renders and matches snapshot', () => {
     const renderedNav = shallow(
       <InteractiveNav
-        report={[FakeInteractiveReport]}
```

Comment:
We don't know why it was being passed as an array?

Reply:
looks like it was a convenience to make the recursive step of parseNavSelection simpler, since it recurses each time into the entries array. But representing the report by a single element is much clearer IMO. Having a report represented by more than one root nodes makes no sense.